crates/egui/src/context.rs

```diff
@@ -328,11 +328,11 @@ impl Context {
         sense: Sense,
         enabled: bool,
     ) -> Response {
-        let gap = 0.5; // Just to make sure we don't accidentally hover two things at once (a small eps should be sufficient).
+        let gap = 0.1; // Just to make sure we don't accidentally hover two things at once (a small eps should be sufficient).
 
         // Make it easier to click things:
         let interact_rect = rect.expand2(
-            (0.5 * item_spacing - Vec2::splat(gap))
+            (0.1 * item_spacing - Vec2::splat(gap))
```

Comment:
The point of expanding by half of the item spacing is to remove the dead space between items. The default `item_spacing.x` is 8, so we have 8 points of space between buttons. But it is nice to be able to click the four pixels to the left or right of a button and still hit the button.

In other words: your change here will make it much _harder_ to hit a button.

```suggestion
            (0.5 * item_spacing - Vec2::splat(gap))
```

Reply:
Ah I see 👍